transformation/Makefile

```diff
@@ -1,6 +1,5 @@
-DBS = "bq/"
 LIBRARY = "library/"
-ALL_SUBFOLDERS = $(DBS) $(LIBRARY)
+ALL_SUBFOLDERS = $(LIBRARY) "bq/" "sf/"
```

Comment:
This has no "sf" for now. Don't know if it will give problems when cleaning. Nice that you noticed that we were missing it in the others.

Reply:
No problem with this